backend/cli.py

```diff
@@ -272,7 +272,7 @@ async def clean_stale_media():
 
 @app.command()
 @coroutine
-async def full_setup(popularity: float = 1, chunk_size: int = 25000):
+async def full_setup(popularity: float = 1, chunk_size: int = 5000):
```

Comment:
Should make the chunk_size 500 like in our crontab?

Also, please update the justfile with whatever amount we go with

Reply:
Wait, this is from an already merged PR I think - Is it a rebase derp?